docker/docker-compose-core.yml

```
@@ -68,7 +68,7 @@ services:
   # ICUBAM back-office service
   app-backoffice:
     build:
-      context: "${ICUBAM_COMPOSE_CONTEXT:-.}"
+      context: "${ICUBAM_COMPOSE_CONTEXT:-..}"
```

See comment below about the mount
```suggestion
      context: "${ICUBAM_COMPOSE_CONTEXT:-.}"
```